Dockerfile

```diff
@@ -1,17 +1,19 @@
 ARG GOVERSION=1.17
 ARG GOARCH
 FROM golang:${GOVERSION} as builder
+USER 65534
```

Comment:
This is the first time I see this pattern during build time, do you have more information or other examples that do the same? I always assumed you would be able to control the buildtime environment from outside and drop privileges there as well. Similar to unprivileged builds in kaniko.

Reply:
i tried to setup the build-stage in the dockerfile the same way as we currently do via the `make build` target.

Within this PR i add the `-u $(USERID):$(USERGROUP)` parameters and i thought it might be a good idea to keep the build-stage configuration "in sync"